src/server/migrations/20201117134811_create_groups_schema.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('group_table', (table) => {
+    table.increments();
+    table.string('title').notNullable();
+    table.datetime('startDate').notNullable();
```

Comment:
Do we need groups to have start/end date? What would be the start/end date for group `class 14`?
These are not study groups but groups a student belongs to.